src/bolt/events/RegisterEventReactionRemove.ts

```diff
@@ -19,40 +20,38 @@ export default function registerEventReactionRemove(app: App) {
             return;
         }
 
-        const {reactionUserId, slackMsgUserId, slackThreadTs} = data;
+        const {pullRequestLink, reactionUserId, slackThreadTs} = data;
 
-        if (slackActions.request.includes(data.reaction)) {
-            const result = await Review.withdraw(data);
-
-            if (!result.user) {
-                await say({
-                    text: `<@${reactionUserId}>, ${result.message}`,
-                    thread_ts: slackThreadTs,
-                });
+        const prisma = new PrismaClient();
+        const codeReview = await prisma.codeReview.findFirst({
+            include: {
+                user: true,
+                reviewers: true,
+            },
+            where: {
+                pullRequestLink,
             }
-            else {
-                const notify = channelNotify[data.slackChannelId] || channelNotify.default;
+        })
```

Comment:
This looks like it's missing a semi-colon, we should get a eslint check in this repo soon :)

Reply:
Good catch, let me see if I can get the more strict rule apply.